orcid/en/appendix-testing-orcid-sandbox.md

```diff
@@ -0,0 +1,50 @@
+# Appendix:  Testing the ORCID Sandbox
+
+Once your Sandbox Client ID and Client Secret have been added to the configuration, you can to test the plugin in the ORCID sandbox environment. The sandbox allows the creation of test user ORCID iD accounts and use of the plugin without having to worry about affecting data on the live (production) ORCID Registry. Once the plugin works as expected in the ORCID sandbox environment, you can obtain credentials and move to the ORCID production environment and go live with the plugin.
+
+
+To test that the plugin is working correctly in the sandbox:
+
+#### Create a Sandbox ORCID iD
```

Comment:
This should be a level 2 heading so that heading levels are not being skipped. Others below may also need to be adjusted.

Reply:
Thanks for catching that. Should be addressed as part of recent commits.